tools/ci-ts/test-helpers/chainlink-cli.ts

```diff
@@ -0,0 +1,40 @@
+import { ethers } from 'ethers'
+import execa from 'execa'
+import { JobSpec, RunResult } from '../../../operator_ui/@types/operator_ui'
+
+const API_CREDENTIALS_PATH = '/run/secrets/apicredentials'
+
+export async function login(): Promise<void> {
+  await execa('chainlink', ['admin', 'login', '--file', API_CREDENTIALS_PATH])
+}
+
+export async function getJobs(): Promise<JobSpec[]> {
+  const { stdout } = await execa('chainlink', ['-j', 'jobs', 'list'])
+  return JSON.parse(stdout) as JobSpec[]
+}
+
+export async function getRunResults(): Promise<RunResult[]> {
+  const { stdout } = await execa('chainlink', ['-j', 'runs', 'list'])
+  return JSON.parse(stdout) as RunResult[]
+}
+
+export async function createJob(jobSpec: string): Promise<JobSpec> {
+  const { stdout } = await execa('chainlink', ['-j', 'jobs', 'create', jobSpec])
+  return JSON.parse(stdout) as JobSpec
+}
+
+export async function archiveJob(jobId: string): Promise<void> {
+  await execa('chainlink', ['-j', 'jobs', 'archive', jobId])
+}
+
+// interface for the data describing the CL node's keys
+interface KeyInfo {
```

Comment:
Nit: We should be using JSDoc style comments. If you're using vscode you can [automatically generate templates](https://code.visualstudio.com/docs/languages/javascript#_jsdoc-support) in your editor.